internal/sirius/date_string.go

```diff
@@ -5,34 +5,41 @@ import (
 	"errors"
 	"fmt"
 	"strings"
+	"time"
 )
 
 // DateString is a date in the format "YYYY-MM-DD" that will unmarshal from and
 // marshal to the Sirius format of "DD/MM/YYYY"
 type DateString string
 
+var formats = []string{
+	"2006-01-02T15:04:05-07:00", // format from payments table
+	"02/01/2006",
+}
```

Comment:
I don't think the alternative format is needed any more? If it is, I'd suggest using `time.Time` for the struct field instead.

Reply:
Yeah you're right, i'll revert the whole file back as my changes aren't needed anymore